pkg/provisioner/block/block.go

```diff
@@ -171,7 +178,7 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions, ad *i
 	}
 
 	if value, ok := options.PVC.Annotations[ociVolumeBackupID]; ok {
-		glog.Infof("Creating volume from backup ID %s", value)
+		logger.With("volumeBackupOCID", value).Info("Creating volume from backup.")
```

Comment:
Should re-assign logger here adding the `"volumeBackupOCID"` to later logs as well.